docs/deployment/landing_page.rst

```diff
@@ -265,13 +265,15 @@ logging. However you still need to make sure no other assets get logged!
     SetEnvIf Request_URI "^/securedrop($|(\/.*))" dontlog
     CustomLog logs/access_log common env=!dontlog
 
-In nginx, logging can be disabled like so:
+In nginx, logging can be disabled by adding the following directives within the
+*Landing Page* ``location {}`` block:
```

Comment:
LGTM based on, can be applied to the location block:
- https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log
- https://docs.nginx.com/nginx/admin-guide/monitoring/logging/